library/core/src/macros/panic.md

```diff
@@ -9,7 +9,7 @@ tests. `panic!` is closely tied with the `unwrap` method of both
 `panic!` when they are set to [`None`] or [`Err`] variants.
 
 When using `panic!()` you can specify a string payload, that is built using
-the [`format!`] syntax. That payload is used when injecting the panic into
+the [`format!` syntax]. That payload is used when injecting the panic into
```

Comment:
suggestion: maybe just say 
```suggestion
the [formatting syntax]. That payload is used when injecting the panic into
```
rather that mentioning a particular macro?  That avoids a conversation about whether it's "the `format!` syntax" or "the `format_args!` syntax" or ...